examples/website/carto-sql/app.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import {render} from 'react-dom';
+import {StaticMap} from 'react-map-gl';
+import DeckGL from '@deck.gl/react';
+import {CartoSQLLayer, setDefaultCredentials} from '@deck.gl/carto';
+
+// Set your mapbox token here
+const MAPBOX_TOKEN = process.env.MapboxAccessToken; // eslint-disable-line
+
+const INITIAL_VIEW_STATE = {
+  latitude: 40.7368521,
+  longitude: -73.9936065,
+  zoom: 11,
+  pitch: 60,
+  bearing: 0
+};
+
+// Colors for the breaks of the polygon layer
+const POLYGON_COLORS = {
+  COLOR_1: [225, 83, 131],
+  COLOR_2: [241, 109, 122],
+  COLOR_3: [250, 138, 118],
+  COLOR_4: [255, 166, 121],
+  COLOR_5: [255, 194, 133],
+  COLOR_6: [255, 221, 154],
+  OTHER: [254, 246, 181]
+};
+
+export default function App({
+  index = 'txn_amt',
+  industry = 'ret',
+  viewState = INITIAL_VIEW_STATE,
+  mapStyle = 'https://basemaps.cartocdn.com/gl/dark-matter-gl-style/style.json'
+}) {
+  setDefaultCredentials({
```

Comment:
Take this out of `App()` and position on the top of this file, so that users know to change it when copied.